docs/source/install/index.rst

```diff
@@ -54,6 +54,7 @@ Once it completes successfully, you will see the following output:
     Ubuntu 14.04 / 16.04 <deb>
     RHEL 7 / CentOS 7 <rhel7>
     RHEL 6 / CentOS 6 <rhel6>
+    Ansible Playbooks <ansible>
```

Comment:
My initial idea was to keep it short as possible and refer to GitHub repo.

But I can bring enough content from the `ansible-st2` README, - it's not a problem. Including more detailed installation examples, requirements and configuration vars.